SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRumBuilder.swift

```diff
@@ -0,0 +1,149 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+@objc public class SplunkRumBuilder: NSObject {
```

Comment:
Will we be documenting these build funcs with some description of what they do?

Reply:
Like dev docs? Those can live in the repo—if they're for developing the distro.

Reply:
Right like the dev docs seen here on the android builder https://github.com/signalfx/splunk-otel-android/blob/main/splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java

I think it is useful for users of the lib to see what the function / property does right in the IDE

Reply:
I don't know. We need a RUM design overlord to decide. :)

Reply:
Btw. All my comments are in relation to a task I am working on - document differences between iOS and Android. Thanks for your input.